### ISSUE TITLE
Drag n drop  problem in Email bigger then screen height (Mautic 3.3 /4 beta)

### ISSUE THREAD

**Issue by DevepNoName** [mautic/mautic#10211]
Thanks for this great framework.

Im using the classic Email editor (not grape) and the Vibrant template (but really could be using other), but when i drag something from the Slot Types, ou Section Types i cant drag n drop for the correct position if the mail its bigger then the screen height.
And isnt scrolling the Email down.
Please see this video: https://youtu.be/0np8r4LOZ4U
Thanks

## Bug Description
The more details the better...

| Q   | A
| --- | ---
| Mautic version | 3.3 / 4 beta
| PHP version | 7.4.18
| Browser | Chrome

### Steps to reproduce
1. Select Mautic classic Email editor > Email > Vibrant (any template with content bigger then the screen height)
2. Drag  Slot Types, ou Section Types to the template
 
### Log errors

[//]: # ( Invisible comment:
Please check for related errors in the latest log file in [mautic root]/app/log/ and/or the web server's logs and post them here. Be sure to remove sensitive information if applicable. )

**Comment by RCheesley** [comment]
Hi there, 

Thanks for flagging this UX issue.

I don't think that we will be making any UX fixes to the legacy builder once Mautic 4 is out as it is being deprecated in favour of the GrapesJS builder and will be removed entirely in Mautic 5.

As a result I am going to close this issue down as a 'won't fix'.

**Comment by DevepNoName** [comment]
Hi, thanks for input, 

gonna give you my feedback, because i think could help somehow.

I was a big fan of GrapeJs, but after using it with Mautic i think would bring more problems then solutions.

I made like 20 componentes (html components: they are HTML verified newslsetters components), for GrapeJs, and when i drag them to Mautic its a big mess. GrapeJs adds a lots of unnecessary code (is putting divs, when i really dont want divs - and shouldnt be used divs in newsletters) - i also tried with the "editable = false"

But the bigger problem, is after you save the email, and open the editor again, its impossible the drag to the right place:
 my structure is

```
<table>component 1</table>
<table>component 2</table>
<table>component 3</table>
```

and after save and open to edit its impossible to do this:
```
<table>component 1</table>
<table>component 2</table>
<table>component 2a</table>
<table>component 3</table>
```

will always be like this:
```
<table>component 1</table>
<table>component 2<table>component 2a</table></table>
<table>component 3</table>
```
and im using data-gjs-droppable="false", but after reload that tag its deleted.

Another problem, is if you deleted a component, grapejs doesnt delete everything, some code stays in the newsletter.

that why i  started to use the  legacy builder, for this use case, its much better. 
I created the same components in the legacy, and the code its clean, and everthyng works. Just had the drag problem above.

I also tried using GrapeJs (mjml), but cant keep the formating as i wanted.

If anyone its interessed in solving i can put a bounty, and i aprecciate that.
Thanks
